sound/soc/sof/intel/hda-loader.c

```diff
@@ -303,6 +303,7 @@ int hda_dsp_cl_boot_firmware_iccmax(struct snd_sof_dev *sdev)
 	struct hdac_ext_stream *iccmax_stream;
 	struct hdac_bus *bus = sof_to_bus(sdev);
 	struct firmware stripped_firmware;
+	struct snd_dma_buffer dmab;
```

Comment:
dmab_bdl

Reply:
Right, what `bdl` stands for? Does it really matter how it is called if it is only used within this function?

Reply:
Buffer Descriptor List. It's useful to differentiate from the actual DMA buffer.

Reply:
I see, the comments are not helping with that distinction, I'll do the rename.